backend/src/main/java/ai/verta/modeldb/artifactStore/storageservice/s3/S3Service.java

```diff
@@ -124,6 +124,52 @@ private Boolean doesObjectExist(String bucketName, String path) throws ModelDBEx
     }
   }
 
+  @Override
+  public GetUrlForArtifact.Response generatePresignedUrlForTrial(
+      String s3Key, String method, long partNumber, String uploadId) throws ModelDBException {
+
+    if (partNumber != 0) {
```

Comment:
this should change to 1 based on the way client checks things
https://github.com/VertaAI/modeldb/blob/master/client/verta/verta/_tracking/experimentrun.py#L272